analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/ProgramState.cs

```diff
@@ -213,7 +213,7 @@ private ProgramState ResetFieldConstraints(Func<IFieldSymbol, bool> predicate)
             return state;
         }
 
-        [Conditional("DEBUG")]
+        [Conditional("DEBUG_TurnedOff")]
```

Comment:
What about this, so the value makes some sense meanwhile for manual configuration?
```suggestion
        [Conditional("DEBUG_CHECK_CONSISTENCY")]
```

Reply:
I think this is more confusing than helpful as long as we do not have such a configuration. We could add a `// #define DEBUG_CHECK_CONSISTENCY` right before the conditional, but this would be just a more complicated version of what I have.

Reply:
You can add comment explaining the intended usage. And that is should also go away soon